modules/rosa-sdpolicy-instance-types.adoc

```diff
@@ -13,20 +13,13 @@ endif::[]
 = Instance types
 
 ifdef::rosa-with-hcp[]
-All {hcp-title} clusters require a minimum of 2 worker nodes. All {hcp-title} clusters support a maximum of 500 worker nodes. Shutting down the underlying infrastructure through the cloud provider console is unsupported and can lead to data loss.
-
-[NOTE]
-====
-Customers running {hcp-title} 4.14.x and 4.15.x clusters require a minimum z-stream version of 4.14.28 or 4.15.15 and greater to scale to 500 worker nodes. For earlier versions, the maximum is 90 worker nodes. 
-====
-
+All {hcp-title} clusters require a minimum of 2 worker nodes. Shutting down the underlying infrastructure through the cloud provider console is unsupported and can lead to data loss.
```

Comment:
What's the background to include the statement regarding shutting down is unsupported. It seems misplaced in this topic. We should remove that.

Reply:
I will look into this.